pkg/expressions/stdlib/funcs.go

```diff
@@ -16,35 +16,35 @@ var StandardFunctions = map[string]KeyBuilderFunction{
 	"isint": KeyBuilderFunction(kfIsInt),
 	"isnum": KeyBuilderFunction(kfIsNum),
 
-	// Arithmatic
-	"sumi":  arithmaticHelperi(func(a, b int) int { return a + b }),
-	"subi":  arithmaticHelperi(func(a, b int) int { return a - b }),
-	"multi": arithmaticHelperi(func(a, b int) int { return a * b }),
-	"divi":  arithmaticHelperi(func(a, b int) int { return a / b }),
-	"modi":  arithmaticHelperi(func(a, b int) int { return a % b }),
-	"maxi": arithmaticHelperi(func(a, b int) int {
+	// Arithmetic
+	"sumi":  ArithmeticHelperi(func(a, b int) int { return a + b }),
```

Comment:
the function names here need to remain lower-case `A`. Go uses capital letters to designate a public function, and this should remain private